sample/src/main/kotlin/com/uber/autodispose/recipes/subscriberproxies.kt

```diff
@@ -0,0 +1,54 @@
+package com.uber.autodispose.recipes
+
+import com.uber.autodispose.*
+import io.reactivex.disposables.Disposable
+import io.reactivex.exceptions.OnErrorNotImplementedException
+import io.reactivex.plugins.RxJavaPlugins
+
+private val onNextStub: (Any) -> Unit = {}
```

Comment:
Let's add a file doc above this that explains how this is an example of how you can tack on extension functions to the proxy interface

```kotlin
/*
 * Stuff!
 */
```